Library/Formula/zurl.rb

```diff
@@ -0,0 +1,21 @@
+class Zurl < Formula
+  homepage "https://github.com/fanout/zurl"
+  url "http://packages.fanout.io/source/zurl-1.3.1.tar.bz2"
+  sha1 "8a04ad092a4fd9b174a352b910c190466fc39eb9"
+
+  depends_on "pkg-config" => :build
+  depends_on "curl"
```

Comment:
Does that apply to all OS X versions? Yosemite ships ` curl 7.37.1 ` for example, which is pretty new.